.pre-commit-config.yaml

```diff
@@ -1,27 +1,29 @@
 ---
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v3.1.0
+    rev: v4.4.0
     hooks:
       - id: trailing-whitespace
       - id: check-added-large-files
       - id: check-docstring-first
       - id: check-merge-conflict
       - id: end-of-file-fixer
   - repo: https://github.com/ambv/black
-    rev: 21.12b0
+    rev: 22.12.0
     hooks:
       - id: black
         language_version: python3.7
   - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v0.910-1
+    rev: v0.991
     hooks:
       - name: Run mypy static analysis tool
         id: mypy
         args: [--no-warn-unused-ignores, --ignore-missing-imports]
         files: src
-  - repo: https://github.com/flakehell/flakehell/
-    rev: v.0.9.0
+        additional_dependencies:
+          - pydantic
+  - repo: https://github.com/flakeheaven/flakeheaven
+    rev: 3.2.1
     hooks:
-      - name: Run flakehell static analysis tool
-        id: flakehell
+      - name: Run flakeheaven static analysis tool
+        id: flakeheaven
```

Comment:
I replaced flakehell with flakeheaven here, as it seems like flakehell is not maintained well, and also flakeheaven was already used in pyproject.toml